pkg/goods/support/host-support-bundle.yaml

```diff
@@ -5,116 +5,189 @@ metadata:
-      outcomes:
-      - warn:
-          when: "< 2G"
-          message: At least 2G of memory is recommended
-      - pass:
-          message: The system has at least 2G of memory
-  - diskUsage:
-      checkName: Root disk usage
-      collectorName: root-disk-usage
-      outcomes:
-      - fail:
-          when: "total < 40Gi"
-          message: The disk containing directory / has less than 40Gi of total space
-      - warn:
-          when: "used/total > 80%"
-          message: The disk containing directory / is more than 80% full
-      - warn:
-          when: "available < 10Gi"
-          message: The disk containing directory / has less than 10Gi of disk space available
-      - pass:
-          message: The disk containing directory / has sufficient space
-  - diskUsage:
-      checkName: OpenEBS disk usage
-      collectorName: openebs-disk-usage
-      outcomes:
-      - fail:
-          when: "total < 40Gi"
-          message: The disk containing OpenEBS volumes has less than 40Gi of space
-      - warn:
-          when: "used/total > 80%"
-          message: The disk containing OpenEBS volumes is more than 80% full
-      - warn:
-          when: "available < 10Gi"
-          message:  The disk containing OpenEBS volumes has less than 10Gi of disk space available
-      - pass:
-          message: The disk containing directory OpenEBS volumes has sufficient space
-  - textAnalyze:
-      checkName: Kubernetes API probing
-      fileName: host-collectors/run-host/k0s-status.txt
-      regex: 'Kube-api probing successful: true'
-      outcomes:
-      - fail:
-          when: "false"
-          message: Kubernetes API probing is reporting a failure
-      - pass:
-          when: "true"
-          message: Kubernetes API probing is reporting success
-  - textAnalyze:
-      checkName: NetworkManager managing calico interfaces
-      fileName: host-collectors/run-host/network-manager-logs.txt
-      regex: 'device .*cali.+: state change: config'
-      outcomes:
-      - fail:
-          when: "true"
-          message: NetworkManager seems to be managing calico interfaces
-      - pass:
-          when: "false"
-          message: NetworkManager isn't managing calico interfaces
-  - hostServices:
-      checkName: "Local Artifact Mirror"
-      outcomes:
-      - fail:
-          when: "local-artifact-mirror != active"
-          message: Local Artifact Mirror isn't active
-      - pass:
-          when:  "local-artifact-mirror = active"
-          message: Local Artifact Mirror is active
+    - ipv4Interfaces:
+        outcomes:
+          - fail:
+              when: 'count == 0'
+              message: No IPv4 interfaces detected
+          - pass:
+              when: 'count >= 1'
+              message: IPv4 interface detected
+    - memory:
+        checkName: Amount of Memory
+        outcomes:
+          - warn:
+              when: '< 2G'
+              message: At least 2G of memory is recommended
```

Comment:
Moving forward with EC, we're going to try not having warnings. Either something is important enough to Fail or it isn't. Choose what you think is a fair solution here.

Reply:
:thumbsup: